dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/DSpaceRestRepository.java

```diff
@@ -97,7 +98,7 @@
     @Override
     /**
      * Return a specific REST object
-     * 
+     *
```

Comment:
I'd consider this very low priority right now.  Not sure if it's easy to check for in code style (and even if it is, it might take significant effort to align all comments).  So, I'm resolving this comment.